app/controllers/users_controller.rb

```diff
@@ -118,7 +121,12 @@ def correct_user
 
     # Confirms administrator
     def admin_user
-      redirect_to(root_url) unless current_user.privilege == "admin"
+      current_user.privilege == "admin"
+    end
```

Comment:
should this be changed back to what is was?
-      redirect_to(root_url) unless current_user.privilege == "admin"